### DISCUSSION TITLE
How do i actually start the desktop app?

### DISCUSSION THREAD

**Discussion by Vito0208** [Shabinder/SpotiFlyer#71] [Q&A] [answered]
I am new to github, but I found how to start the desktop app, but I just don't know what to do with that command that is listed under "Starting the desktop application". Please help.

**Comment by Shabinder** [comment, marked as answer]
I understand the confusion some might have, actually as stated earlier desktop app is still in Beta , and for now I haven't made any packaged Distributables,

However if you want to try out the app, Clone / Download SpotiFlyer Repo and open terminal and cd into cloned directory ,then run the command as shown in readme.
after executing that command , gradle will be downloaded and desktop app will be built and then run, I will repeat again that Desktop App is still in preview mode and has some bugs and needs some finishing.

**Reply by Vito0208** [reply]
![Snimka zaslona 2021-03-22 233509](https://user-images.githubusercontent.com/38729403/112066779-43c7ec00-8b67-11eb-8aa9-c9acd9aa1205.png)

**Reply by Shabinder** [reply]
> ![Snimka zaslona 2021-03-22 233509](https://user-images.githubusercontent.com/38729403/112066779-43c7ec00-8b67-11eb-8aa9-c9acd9aa1205.png)

yeah, one more thing , you need to have jdk 15 installed too.

**Reply by Vito0208** [reply]
thanks you very much
